src/platform/telemetry/index.ts

```
@@ -2,41 +2,57 @@
+  if (!IS_CLOUD_BUILD) return
+  if (_initPromise) return _initPromise
 
-// Singleton instance
-let _telemetryProvider: TelemetryProvider | null = null
+  _initPromise = (async () => {
+    const [
+      { TelemetryRegistry },
+      { MixpanelTelemetryProvider },
+      { GtmTelemetryProvider }
+    ] = await Promise.all([
+      import('./TelemetryRegistry'),
```

ESLint

[error] Unable to resolve path to module './TelemetryRegistry'.

(import-x/no-unresolved)